src/format/locales.rs

```diff
@@ -31,3 +31,115 @@ pub(crate) fn d_t_fmt(locale: Locale) -> &'static str {
 pub(crate) fn t_fmt(locale: Locale) -> &'static str {
     locale_match!(locale => LC_TIME::T_FMT)
 }
+
+macro_rules! expand_ampm {
+    ($item:ident) => {
+        pure_rust_locales::$item::LC_TIME::T_FMT_AMPM
+    };
+}
+
+/// hardcoded `locale_match!` fallback for `T_FMT_AMPM`.
+///
+/// it takes variable number of possible locales that have `LC_TIME::T_FMT_AMPM` field.
+/// it will then match the locale and return its `LC_TIME::T_FMT_AMPM` field.
+/// fallbacks to en_US if the field is an empty string.
+///
+/// ```rs
```

Comment:
This docstring code should be ignored. Also, I don't think `rs` is valid

    /// ```rust,ignore